api/src/prescription/campaign-participation/application/pole-emploi-route.js

```diff
@@ -10,7 +10,7 @@ const register = async function (server) {
       method: 'GET',
       path: '/api/pole-emploi/envois',
       config: {
-        auth: 'jwt-pole-emploi',
+        auth: { strategy: 'jwt-pole-emploi', acess: { scope: 'pole-emploi-participants-result' } },
```

Comment:
Ça fait planter l'API ;)
```suggestion
        auth: { strategy: 'jwt-pole-emploi', access: { scope: 'pole-emploi-participants-result' } },
```